### DISCUSSION TITLE
how to push an object to a json array using supabase js library?

### DISCUSSION THREAD

**Discussion by saman1324** [supabase/supabase#6008] [Questions] [answered]
hello. i have a json array column. it is a little difficult to push an object to it. i should get that json array, push an object to it and update that column. i should send two request and its not what i want because it makes some problem in my web app.
my question is: is there an other way, an easier one, to push an object to a json array using supabase library? for example sending an object that i want to push to the json array to a method of supabase library as an argument and then supabase push that object to the json array. is it possible?
thanks for helping.

**Comment by steve-chavez** [comment, marked as answer]
You need [rpc](https://supabase.com/docs/reference/javascript/rpc) for now. See this previous discussion https://github.com/supabase/supabase/discussions/1570
